includes/definitions/webpower-smart2.yaml

```diff
@@ -0,0 +1,15 @@
+os: webpower-smart2
+text: WebPower-smart2
+group: ups
+icon: generic
```

Comment:
Can we not get a logo for this?

Reply:
> Can we not get a logo for this?

hi @laf 
there is at least 3 manufacturer who use USHA as base for their cards
Falcon, Alpha, Eaton ...
this one i have at hand, have NO manufacturer printed on PCB :(
and if you look at old webpower definition
```
/opt/librenms/includes/definitions/webpower.yaml
os: webpower
text: WebPower
type: power
rfc1628_compat: true
```
there is also no manufacturer logo